### PR TITLE
Move e2e tests to PR check and prod-preview periodic tests

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Flaky test suite runs only against production, which is not compatible with current state of tests. Moving the e2e tests as a part of PR check instead.

### CODE REVIEW

File: functional-tests/devscripts/run_tests.sh [hunk 1]
```diff
@@ -113,8 +113,29 @@ if [[ "$PR_CHECK_BUILD" == "true" ]]; then
 	           -e "OPENSHIFT_URL=$OC_CLUSTER_URL" \
 	           -e "OPENSHIFT_USERNAME=$RH_CHE_AUTOMATION_CHE_PREVIEW_USERNAME"  \
 	           -e "OPENSHIFT_PASSWORD=$RH_CHE_AUTOMATION_CHE_PREVIEW_PASSWORD" \
-           quay.io/openshiftio/rhchestage-rh-che-functional-tests-dep
-    RESULT=$?
+	       quay.io/openshiftio/rhchestage-rh-che-functional-tests-dep
+	RESULT=$?
+	    
+	mkdir screens
```
Comment (rhopp) [inline functional-tests/devscripts/run_tests.sh]: Wouldn't `report` be more in-line with what is used, when running tests locally?

Comment (Katka92) [reply functional-tests/devscripts/run_tests.sh]: Hm, good point, report should be better then screens. Will fix that.

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
[test]

**Comment by Katka92** [comment]
Typescript tests passed, old tests are failing on ` Timed out receiving message from renderer ` - known issue. Merging.

**Pull request merged**